src/m365/aad/commands/approleassignment/approleassignment-add.ts

```diff
@@ -179,19 +179,19 @@ class AadAppRoleAssignmentAddCommand extends GraphCommand {
         throw `The resource '${args.options.resource}' does not have any application permissions available.`;
       }
 
-      // search for match between the found app roles and the specified scope option value
-      for (const scope of args.options.scope.split(',')) {
+      // search for match between the found app roles and the specified scopes option value
+      for (const scope of args.options.scopes.split(',')) {
         const existingRoles = appRolesFound.filter((role: AppRole) => {
           return role.value.toLocaleLowerCase() === scope.toLocaleLowerCase().trim();
         });
 
         if (!existingRoles.length) {
-          // the role specified in the scope option does not belong to the found service principles
+          // the role specified in the scopes option does not belong to the found service principles
           // throw an error and show list with available roles (scopes)
           let availableRoles: string = '';
           appRolesFound.map((r: AppRole) => availableRoles += `${os.EOL}${r.value}`);
 
-          throw `The scope value '${scope}' you have specified does not exist for ${args.options.resource}. ${os.EOL}Available scopes (application permissions) are: ${availableRoles}`;
+          throw `The scopes value '${scope}' you have specified does not exist for ${args.options.resource}. ${os.EOL}Available scopes (application permissions) are: ${availableRoles}`;
```

Comment:
Actually, I think the first one was ok. It should be `scope` as the specified single `scope` did not match any of the allowed once right? 🤔

```suggestion
          throw `The scope value '${scope}' you have specified does not exist for ${args.options.resource}. ${os.EOL}Available scopes (application permissions) are: ${availableRoles}`;
```